src/main/java/net/sf/jabref/gui/FXAlert.java

```diff
@@ -89,6 +96,42 @@ public FXAlert(AlertType type) {
         fxDialogWindow.setOnHiding(evt -> setSwingWindowsEnabledAndFocusable(true));
 
         fxDialogWindow.setOnCloseRequest(evt -> this.close());
+
+        fxDialogWindow.getScene().setOnKeyPressed(evt -> {
+            KeyCombination combination = Globals.getKeyPrefs().getKeyCombinationFX(KeyBinding.CLOSE_DIALOG);
+            if (checkKeyCombinationEquality(combination, evt)) {
+                fxDialogWindow.close();
+            }
+        });
+    }
+
+    /**
+     * Check if the given keyCombination equals the given keyEvent
+     *
+     * @param combination as KeyCombination
+     * @param keyEvent as KeEvent
+     * @return true if matching, else false
+     */
+    private boolean checkKeyCombinationEquality(KeyCombination combination, KeyEvent keyEvent) {
```

Comment:
I would prefer if you move this code to the `KeyBindingPreferences` class. Than you can write something like `if(Globals.getKeyPrefs().checkKeyCombinationEquality(KeyBinding.CLOSE_DIALOG, event)`